diff --git a/lib/utilities/build-bundles-async/build-bundles-async.js b/lib/utilities/build-bundles-async/build-bundles-async.js
index e44dbd50553277e2aeccf90a0cecb28acb1aea42..bb02a522c756d95484c2efb47e461a7660d5a03a 100644
--- a/lib/utilities/build-bundles-async/build-bundles-async.js
+++ b/lib/utilities/build-bundles-async/build-bundles-async.js
@@ -5,7 +5,6 @@ import { globby } from 'globby';
 import indentString from 'indent-string';
 import { importFresh } from '../import-fresh.js';
 import { esbuildCssModulesPlugin } from './esbuild-css-modules-plugin.js';
-import { esbuildPreactCompatPlugin } from './esbuild-preact-compat-plugin.js';
 export async function buildBundlesAsync(options) {
     const { config, minify, outputDirectory } = options;
     await Promise.all([
@@ -91,7 +90,7 @@ async function buildUiBundleAsync(options) {
             logLevel: 'silent',
             minify,
             outfile: resolve(outputDirectory, constants.build.pluginUiFilePath),
-            plugins: [esbuildPreactCompatPlugin(), esbuildCssModulesPlugin(minify)],
+            plugins: [esbuildCssModulesPlugin(minify)],
             stdin: {
                 contents: js,
                 resolveDir: process.cwd()